src/test/java/bio/terra/service/snapshotbuilder/utils/CriteriaQueryBuilderTest.java

```diff
@@ -243,24 +243,26 @@ void generateRollupCountsQueryForCriteriaGroupsList(SqlRenderContext context) {
                             .meetAll(true)
                             .mustMeet(true))));
     // FIXME: is query correct? It doesn't contain the concept IDs 11 and 10.
```

Comment:
is this the fix to this problem? or is there a separate ticket to address this problem? I still don't see concept ids 10 or 11 in the query

Reply:
This comment wasn't included in this PR, I'll give it a look and see if it's a ticket.

Reply:
Okay, I don't see one so I will file a new bug!